src/Auth0.php

```diff
@@ -246,28 +246,28 @@ public function decode(
         ?string $tokenNonce = null,
         ?int $tokenMaxAge = null,
         ?int $tokenLeeway = null,
-        ?int $tokenNow = null
+        ?int $tokenNow = null,
+        ?int $tokenType = null
```

Comment:
@evansims updated this PR to add the missing `$tokenType` parameter for decoding.